testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -397,3 +397,81 @@ def test_table_atoms(self, h, normalized_timeseries, reference_table):
         # https://github.com/MDAnalysis/mdanalysis/issues/801)
         for name, ref in reference_table.items():
             assert_array_equal(h.table.field(name), ref, err_msg="resname for {0} do not match (Issue #801)")
+
+
+class TestHydrogenBondAnalysisTIP3PHeavyPBC(object):
+    @staticmethod
+    @pytest.fixture()
+    def universe():
```

Comment:
add `scope="class"` so that it only gets run once – works fine if data are only read.

Reply:
I've added this.